docs/content/latest/quick-start/build-apps/go/ysql-pg-ssl.md

```diff
@@ -0,0 +1,334 @@
+}
+
+func main() {
+    var sslMode = ""
+    var ssl = os.Getenv("PGSSLMODE")
+    if ssl != "" {
+        sslMode = "?sslmode=" + ssl
+    }
+
+    url := fmt.Sprintf("postgres://%s:%s@%s:%d/%s%s",
+                      user, password, host, port, dbname, sslMode)
+    opt, errors := pg.ParseURL(url)
+    if errors != nil {
+        log.Fatal(errors)
+    }
+
+    CAFile := os.Getenv("PGSSLROOTCERT")
+    if (CAFile != "") {
+        CACert, err2 := ioutil.ReadFile(CAFile)
+        if err2 != nil {
+            log.Fatal(err2)
+        }
+
+        CACertPool := x509.NewCertPool()
+        CACertPool.AppendCertsFromPEM(CACert)
+
+        tlsConfig := &tls.Config{
+          RootCAs:            CACertPool,
+          ServerName:         host,
+        }
+        opt.TLSConfig = tlsConfig
+    }
+    db := pg.Connect(opt)
+
+    defer db.Close()
+
+    model := (*Employee)(nil)
+    err := db.Model(model).DropTable(&orm.DropTableOptions{
+        IfExists: true,
+    })
+    if err != nil {
+        log.Fatal(err)
+    }
+
+    err = db.Model(model).CreateTable(&orm.CreateTableOptions{
+        Temp: false,
+    })
+    if err != nil {
+        log.Fatal(err)
+    }
+
+    fmt.Println("Created table")
+
+    // Insert into the table.
+    employee1 := &Employee{
+        Name:   "John",
+        Age:    35,
+        Language: []string{"Go"},
+    }
+    _, err = db.Model(employee1).Insert()
+    if err != nil {
+        log.Fatal(err)
+    }
+
+    _, err = db.Model(&Employee{
+        Name:      "Kelly",
+        Age:       35,
+        Language:  []string{"Golang", "Python"},
+    }).Insert()
+    if err != nil {
+        log.Fatal(err)
+    }
+
+    fmt.Println("Inserted data")
+
+    // Read from the table.
+    emp := new(Employee)
+    err = db.Model(emp).
+        Where("employee.id = ?", employee1.Id).
+        Select()
+    if err != nil {
+        log.Fatal(err)
+    }
+    fmt.Printf("Query for id=1 returned: ");
+    fmt.Println(emp)
+}
+```
+
+The `const` values are set to the defaults for a local installation of YugabyteDB. If you are using Yugabyte Cloud, replace the `const` values in the file as follows:
+
+- `host` - The host address of your cluster. The host address is displayed on the cluster Settings tab.
+- `user` - Your Yugabyte database username. In Yugabyte Cloud, the default user is `admin`.
+- `password` - Your Yugabyte database password.
+- `dbname` - The name of the Yugabyte database. The default Yugabyte database name is `yugabyte`.
+
+`port` is set to 5433, which is the default port for the YSQL API.
+
+{{< note title="Note">}}
+If the `password` contains these special characters (#, %, ^), the driver may fail to parse the url. In such a case, use pg.Options() instead of pg.ParseURL() to initialize the Options in `ybsql_hello_world.go`. The standard PG environment variables except `PGPASSWORD` and `PGSSLROOTCERT` are implicitly read by the driver.
+{{< /note >}}
+
+```sh
```

Comment:
Replace the Note with a subhead as follows:

### Using pg.options()

If the password contains these special characters (#, %, ^), the driver may fail to parse the url. In such a case, use pg.Options() instead of pg.ParseURL() to initialize the Options in ybsql_hello_world.go. The standard PG environment variables except PGPASSWORD and PGSSLROOTCERT are implicitly read by the driver.

To use pg.Options(), replace the main function in your file with the following: